inkcut/core/svg.py

```diff
@@ -114,6 +114,10 @@ def parseUnit(string):
         """ Returns userunits given a string representation of units 
         in another system
         """
+
+        if not string:
```

Comment:
I think an `isinstance(string, (int, float))` check would be better. Thoughts?

Reply:
I'm not sure that makes sense to me - that means passing in `42` would return `0`?

It does work (just checked) and I'm not a very opinionated pythonista, so happy to change it if you prefer `isinstance(string, (int, float))` I'll happily update the PR

Reply:
I guess another option might be `if isinstance(string, (int, float)): return string`, but that also seems kind of dodgy

Reply:
Yes, I was thinking then return string.  This way if some attribute has a default of something other than 0 it would still return the correct value if ommitted.

Agree, It might be better to rename the argument to something more generic like `value` or something.